locale/de_DE/submission.xml

```diff
@@ -91,10 +96,16 @@
 	<message key="submission.complete">Freigegeben</message>
 	<message key="submission.incomplete">Warte auf Freigabe</message>
 	<message key="submission.editCatalogEntry">Eintrag</message>
+	<message key="submission.onlineFirst">Online-First</message>
```

Comment:
Do these keys need changing?

Reply:
removed  with  the commit
https://github.com/pkp/omp/pull/687/commits/153406de4b3110ad817aca11161b4ddcbda5d3bf